assets/i18n/qqq.json

```diff
@@ -64,6 +64,9 @@
 	"I18N_ACTION_GUIDE_FOR_TEACHERS": "First button in last section of Oppia's About page targeting guide for teachers page.",
 	"I18N_ACTION_TIPS_FOR_PARENTS": "Second button in last section of Oppia's about page targeting tips for parents and guardians page.",
 	"I18N_ACTION_VISIT_CLASSROOM": "Button below paragraph in first and fourth section in Oppia's About page targeting the classroom.",
+	"I18N_BLOG_DASHBOARD_PAGE_NEW_POST_BUTTON": "Button on Oppia's Blog Dashboard Page. -When user clicks on the button a new blog post is created.",
+    "I18N_BLOG_DASHBOARD_TAB_DRAFTS": "Tab button on Oppia's Blog Dashboard Page. -When user clicks on the button, all the blog post cards for blog posts which are yet to be published are visible.",
```

Comment:
Can you fix the indentation here?

Reply:
Done.

Reply:
Not done, check again. I suggest changing indentation config of vscode from tabs to spaces.

Reply:
weird.
![Screenshot from 2021-07-26 18-53-06](https://user-images.githubusercontent.com/68547101/126995953-975519d2-a7b6-4bf2-9da3-fea30ca03c59.png)
Ill try to fix with git editor.

Reply:
@ashutoshc8101 PLease see changes from all commits.
![Screenshot from 2021-07-26 18-57-40](https://user-images.githubusercontent.com/68547101/126996712-c46a9737-36b6-4609-8e63-028f7cf9a883.png)
Indentation in git to appears in line.

Reply:
Done.